cross/lua/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = lua
+PKG_VERS = 5.3.4
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://www.lua.org/ftp/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+BUILD_DEPENDS = cross/ncurses cross/readline
+
+HOMEPAGE = https://www.lua.org/
+COMMENT = Lua is a powerful, efficient, lightweight, embeddable scripting language. It supports procedural programming, object-oriented programming, functional programming, data-driven programming, and data description.
+LICENSE = MIT
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = myCompile
```

Comment:
Please use more recent naming convention: "lua_custom_compile" and "lua_custom_install" - which helps to understand faster what fails at build !